investpy/__init__.py

```diff
@@ -1233,41 +1317,49 @@ def get_fund_historical_data(fund, country, from_date, to_date, as_json=False, o
     """
```

Comment:
All the changes done not related with `cross_currencies` should not be included since the previous code was properly formatted and the published changes are just modifications on how the data is tabulated or presented to the programmer. So **just the additions related to cross currencies data retrieval will be taken into consideration**, unless changes in any other function improves it.